src/S3/Tests/Unit/Result/GetObjectAclOutputTest.php

```diff
@@ -12,34 +10,46 @@
 
 class GetObjectAclOutputTest extends TestCase
 {
+    public function testGetObjectAclOutput(): void
+    {
+        self::markTestSkipped('Not implemented');
+
+        $response = new SimpleMockedResponse('<ChaneIt/>');
+
+        $result = new GetObjectAclOutput($response, new MockHttpClient());
+
+        // self::assertTODO(expected, $result->getOwner());
+        // self::assertTODO(expected, $result->getGrants());
+        self::assertStringContainsString('change it', $result->getRequestCharged());
+    }
+
     public function testGrantsOutput()
     {
-        $response = new SimpleMockedResponse(<<<XML
-<?xml version="1.0" encoding="UTF-8"?>
-<AccessControlPolicy xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
-    <Owner>
-        <ID>abc</ID>
-        <DisplayName>You</DisplayName>
-    </Owner>
-    <AccessControlList>
-        <Grant>
-            <Grantee xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="CanonicalUser">
+        $response = new SimpleMockedResponse('
+        <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I dont think there can be any whitespace before `<?xml`

Reply:
good catch